### DISCUSSION TITLE
Welcome to RunTime-ToolKit Discussions!

### DISCUSSION THREAD

**Discussion by SwiftExpat** [SwiftExpat/RunTime-ToolKit#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by plumothy** [comment]
When I purchased RunTime Toolkit for Delphi 11, I got very good support from Neil Laskowski at SwiftExPat.

But I am getting no response from SwiftExPat when asking about a version for Delphi 12.

Does anyone know what has happened?
